razzle.config.js

```diff
@@ -42,6 +42,17 @@ module.exports = {
             // externals means files not generated by webpack
             // an alternative would be html-webpack-plugin
             caches: 'all',
+            externals: [
```

Comment:
Do we want to serve fonts here?

Reply:
Update, I also revised how we inject fonts, so we only inject fonts we actually use. I did this here not separately as it clarifies why only some are cached.